ProblemSolvingMentoringSession/Part 0/Return5thElemntOfArrayIfExistsChallenge/Solvings/Return5thElemntOfArrayIfExistsChallenge_Yurchenko.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Return5thElemntOfArrayIfExistsChallenge_Yurchenko.swift
+//  ProblemSolvingMentoringSession
+//
+//  Created by Владислав Юрченко on 28.04.2023.
+//  Copyright © 2023 Almost Engineer. All rights reserved.
+//
+
+import Foundation
+ 
+final class Return5thElemntOfArrayIfExistsChallenge_Yurchenko: Return5thElemntOfArrayIfExistsChallengeProtocol {
+    func solve(input: [Int]) -> Int? {
+        guard !input.isEmpty else { return nil }
+        var result: Int?
+        for item in input {
```

Comment:
Need rework, check how to access item of array by index. You don't need `for in loop for this